src/docs-assembler/navigation.yml

```diff
@@ -37,60 +37,112 @@ toc:
+        # Elasticsearch Java Client
+        # ✅ https://github.com/elastic/elasticsearch-java/blob/main/docs/release-notes/toc.yml
+        - toc: elasticsearch-java://release-notes
+          path_prefix: release-notes/elasticsearch-java-client
+
+        # Elasticsearch JavaScript Client
+        # ✅ https://github.com/elastic/elasticsearch-js/blob/main/docs/release-notes/toc.yml
+        - toc: elasticsearch-js://release-notes
+          path_prefix: release-notes/elasticsearch-javascript-client
+
+        # Elasticserch PHP Client
+        # ✅ https://github.com/elastic/elasticsearch-php/blob/main/docs/release-notes/toc.yml
+        - toc: elasticsearch-php://release-notes
+          path_prefix: release-notes/elasticsearch-php-client
+
+        # Elasticserch Python Client
+        # ✅ https://github.com/elastic/elasticsearch-py/blob/main/docs/release-notes/toc.yml
+        - toc: elasticsearch-py://release-notes
+          path_prefix: release-notes/elasticsearch-python-client
+
+    # Kibana
+    # ✅ https://github.com/elastic/kibana/blob/main/docs/release-notes/toc.yml
+    - toc: kibana://release-notes
+      path_prefix: release-notes/kibana
+
+    # Logstash
+    # ✅ https://github.com/elastic/logstash/blob/main/docs/release-notes/toc.yml
+    - toc: logstash://release-notes
+      path_prefix: release-notes/logstash
+
+    # Beats
+    # ✅ https://github.com/elastic/beats/blob/main/docs/release-notes/toc.yml
+    - toc: beats://release-notes
+      path_prefix: release-notes/beats
+
+    # Fleet and Elastic Agent
+    # ✅ https://github.com/elastic/docs-content/blob/main/release-notes/fleet-elastic-agent/toc.yml
+    - toc: docs-content://release-notes/fleet-elastic-agent
+      path_prefix: release-notes/fleet-elastic-agent
+
+    # Elastic Cloud Hosted
+    # ✅ https://github.com/elastic/cloud/blob/master/docs/release-notes/cloud-hosted/toc.yml
+    - toc: cloud://release-notes/cloud-hosted
+      path_prefix: release-notes/elastic-cloud-hosted
+
+    # Elastic Cloud Serverless
+    # ✅ https://github.com/elastic/docs-content/blob/main/release-notes/elastic-cloud-serverless/toc.yml
+    - toc: docs-content://release-notes/elastic-cloud-serverless
+      path_prefix: release-notes/elastic-cloud-serverless
+
+    # Elastic Cloud Enterprise
+    # ✅ https://github.com/elastic/cloud/blob/master/docs/release-notes/cloud-hosted/toc.yml
+    - toc: cloud://release-notes/cloud-enterprise
+      path_prefix: release-notes/elastic-cloud-enterprise
+
+    # Elastic Cloud on Kubernetes
+    # ✅ https://github.com/elastic/cloud-on-k8s/blob/main/docs/release-notes/toc.yml
+    - toc: cloud-on-k8s://release-notes
+      path_prefix: release-notes/elastic-cloud-kubernetes
+
+    # Elastic Observability
+    # ✅ https://github.com/elastic/docs-content/blob/main/release-notes/elastic-observability/toc.yml
+    - toc: docs-content://release-notes/elastic-observability
+      path_prefix: release-notes/elastic-observability
+      children:
```

Comment:
@KOTungseth we're missing several APM agent release notes:

* `apm-agent-android`
* `apm-agent-java`
* `apm-agent-nodejs`
* `apm-agent-php`
* `apm-agent-python`

Reply:
Moved to https://github.com/elastic/docs-builder/pull/803.